packages/components/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Experimental
```

Comment:
Should this be performance instead of "experimental" or maybe enhancements.

Reply:
I followed the patterns of the other sections which puts all the Tabs stuff under experimental. I don't think there's a precedent for a performance section?

Reply:
It's fine, I don't know much about how they organize stuff either :P